eos-core-depends

```diff
@@ -58,3 +58,94 @@ unoconv
 unrar
 whiptail
 xdg-app
+
+
+# moved here from os-depends due to installer
+chromium-browser
+cups
+eos-social
+eos-speedwagon
+eos-tech-support
+eos-tutorial
+eos-updater
+evince
+fake-hwclock
+file-roller
+fonts-arabeyes
+fonts-arphic-ukai
+fonts-arphic-uming
+fonts-cabinsketch
+fonts-crosextra-caladea
+fonts-crosextra-carlito
+fonts-dejavu
+fonts-dosis
+fonts-droid
+fonts-dustin
+fonts-farsiweb
+fonts-gargi
+fonts-indic
+fonts-kacst
+fonts-khmeros
+fonts-knda
+fonts-lato
+fonts-nanum
+fonts-nanum-coding
+fonts-ocr-a
+fonts-paktype
+fonts-quattrocento
+fonts-roboto
+fonts-sil-andika
+fonts-sil-padauk
+fonts-thai-tlwg
+fonts-tlwg-waree
+fonts-unfonts-core
+fonts-uralic
+fonts-vlgothic
```

Comment:
The fonts are going to need some more analysis, but I think you might want to keep at least fonts-dejavu as it provides a lot of coverage.